recipes/biobox/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "biobox" %}
+{% set version = "1.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Degiacomi-Lab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 749f5a546b4c6e17c369bf487c7ef08e5970f9da6c2f1b790a8e6efbb4804e85
+
+build:
+  number: 0
+  script: |
+        {{ PYTHON }} setup.py build_ext --inplace
+        {{ PYTHON }} setup.py install
```

Pip should handle both calls here.

```suggestion
        script: {{ PYTHON }} -m pip install . -vv
```

---

Please explain how this is an improvement?

---

Generally, it's better practice nowadays to use `pip` instead of calling setup.py directly, as (in general) it doesn't depend so much on package-specific implementations and things potentially going away from the standard library.

I imagine the general intention here to try to make the way this is done across conda-forge as uniform as possible.